src/components/BillCategoryTag/BillCategoryTag.svelte

```diff
@@ -0,0 +1,16 @@
+<script lang="ts">
+	import { Tag } from 'carbon-components-svelte';
+
+	export let isLarge = false;
+	let label = 'ขนส่งสาธารณะ';
+	let tagStyle = 'text-text-10 border-solid bg-white m-0 whitespace-nowrap';
+
+	let className = '';
+	export { className as class };
+</script>
+
+<Tag
+	class="{tagStyle} font-{isLarge ? 'semibold' : 'small'} {className}"
+	style="--tw-bg-opacity: 0; border-width: 1px;"
```

Comment:
If you want to go all in Tailwind, I think you can replace `bg-white` with `bg-white/0` for zero opacity and `border-[1px]`. (Please try first)